python/python-core/src/main/python/_timefold_java_interop.py

```diff
@@ -88,7 +88,9 @@ def init(*args, path: list[str] = None, include_timefold_jars: bool = True) -> N
         path = path + extract_timefold_jars()
     if len(args) == 0:
         args = [get_default_jvm_path()]
-    init(*args, path=path, include_translator_jars=False)
+
+    # JDK 24 bug: Compiler tier 4 (C2) cause a crash at exit
+    init(*args, '-XX:TieredStopAtLevel=3', path=path, include_translator_jars=False)
```

Comment:
Can we do this only when Java is 24?
Also, this cannot be merged, because the tests fail. The failures need to be addressed somehow.

Reply:
Test failures are from the Solver being too slow for integration tests without C2.
As for getting the JVM version, that can only be determined after launching the JVM, at which point, we cannot disable C2 (see https://jpype.readthedocs.io/en/latest/userguide.html#launching-the-jvm).